src/Internal.AspNetCore.Sdk/build/ApiCheck.targets

```diff
@@ -5,10 +5,10 @@
 <Project>
   <Target Name="ApiCheck" Condition=" '$(EnableApiCheck)' == 'true' ">
     <PropertyGroup>
-      <_ApiListingFileSuffix Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">net45.json</_ApiListingFileSuffix>
+      <_ApiListingFileSuffix Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">net46.json</_ApiListingFileSuffix>
```

Comment:
@Eilon - This one is bit interesting.
for .NETFramework tfm we used to use net45.json which is inconsistent now because we upgraded to net46. Though all our baselines say, net45 only. On the other side of world (in netcore/standard) anything non-net framework we used netcore (without exact version) as suffix. Should we change this one to something other than net46? Like netframework & netcore baselines?